libs/common/src/lib/generate-feature-key.ts

```diff
@@ -0,0 +1,5 @@
+import { generateId } from './generate-id';
```

Comment:
FYI just noticed this `generateId` function could return a `SyntaxError` if `Math.random()` returns `0` which is possible ([docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random)). Probability is extremely low but you might want to look into mitigating that in a future release.

Reply:
Good point! yes it is not very probable. But still the id generation could be improved. 

Maybe the nano id implementation is not so bad: https://github.com/ai/nanoid/blob/main/non-secure/index.js#L23-L32

I could imagine to also use shorter ids. Now the length is 10 or 11. 

IDs with a length of 6-8 should be good enough.

Reply:
Created an issue: https://github.com/spierala/mini-rx-store/issues/224